openrtb_ext/imp_gumgum.go

```diff
@@ -1,5 +1,14 @@
 package openrtb_ext
 
+// ExtImpGumGum defines the contract for bidrequest.imp[i].ext.gumgum
+// Either Zone or PubId must be present, others are optional parameters
 type ExtImpGumGum struct {
-	Zone string `json:"zone"`
+	Zone   string  `json:"zone,omitempty"`
+	PubId  float64 `json:"pubId,omitempty"`
```

Comment:
Nitpick: Per Go naming convention, the field names should be "PubID" and "IrisID". The json names are unaffected.